package.json

```diff
@@ -17,8 +17,8 @@
     "deploy:all:production": "yarn deploy:customer1:production && yarn deploy:customer2:production",
     "deploy:all:production:simple": "yarn deploy:customer1:production:simple && yarn deploy:customer2:production:simple",
     "deploy:all:all": "yarn deploy:all && yarn deploy:all:production",
-    "deploy:ci:gha": "yarn vercel:cleanup && yarn vercel:deploy --local-config=vercel.${CUSTOMER_REF}.staging.json",
-    "deploy:ci:gha:production": "yarn vercel:cleanup && yarn vercel:deploy --local-config=vercel.${CUSTOMER_REF}.production.json --prod",
+    "deploy:ci:gha": "yarn vercel:deploy --local-config=vercel.${CUSTOMER_REF}.staging.json",
+    "deploy:ci:gha:production": "CUSTOMER_REF=${CUSTOMER_REF} yarn vercel:deploy --local-config=vercel.${CUSTOMER_REF}.production.json --prod",
```

Comment:
Why `CUSTOMER_REF=${CUSTOMER_REF}` ? Doesn't seem necessary, and it's confusing because the above command doesn't define it.

Reply:
Misstake, I removed it from staging but not production